### PR TITLE
Add wallet-url configuration individually

### PR DESCRIPTION
**Pull request by cookedsteak**
added `WalletURL` in API struct.
modified call method in api.go to call wallet service separately.
```
var targetURL string
if baseAPI == "wallet" && api.WalletURL != "" {
	targetURL = fmt.Sprintf("%s/v1/%s/%s", api.WalletURL, baseAPI, endpoint)
} else {
	targetURL = fmt.Sprintf("%s/v1/%s/%s", api.BaseURL, baseAPI, endpoint)
}
```

Maybe its an optional change for security reasons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-EOS.IO API library for Go
```
Comment (maoueh) [inline README.md]: Revert those changes, I guess they are related to your own fork of the library.

File: api.go [hunk 2]
```diff
@@ -341,6 +342,11 @@ func (api *API) GetProducers() (out *ProducersResp, err error) {
 	return
 }
 
+func (api *API) GetAbiBinToJson(code AccountName, action ActionName, binargs string,) (out interface{}, err error) {
```
Comment (maoueh) [inline api.go]: Do you want to keep this part of this PR?

File: api.go [hunk 3]
```diff
@@ -409,7 +415,13 @@ func (api *API) call(baseAPI string, endpoint string, body interface{}, out inte
 		return err
 	}
 
-	targetURL := fmt.Sprintf("%s/v1/%s/%s", api.BaseURL, baseAPI, endpoint)
+	var targetURL string
+	if baseAPI == "wallet" && api.WalletURL != "" {
```
Comment (maoueh) [inline api.go]: Can the wallet API be located at the same location as the `BaseURL` or it's impossible? If the latter is true (cannot be at same location than BaseURL), than we should probably return an error if somehow is trying to use the `wallet` baseAPI but did not configure correctly `WalletURL`.

File: api.go (hunk 2 above)
Comment (cookedsteak) [reply api.go]: I will add this in another PR.

File: README.md (hunk 1 above)
Comment (cookedsteak) [reply README.md]: I've reverted, sry

**Comment by maoueh** [comment]
What's your use case for this? I discussed this with @abourget and he suggested instead of doing to create two API objects, one for "normal" API and another, configured differently (different "baseURL") for the "wallet" functions.

He also pointer that it's possible to configure the signer with a different API object if that's your use case.

**Comment by maoueh** [comment]
Closing due to lack of activity, don't hesitate to ask for a re-open if you still need something around that PR.

**Pull request closed without merging**